lib/makeWebpackConfig.js

```diff
@@ -36,13 +36,16 @@ module.exports = async (playroomConfig, options) => {
   }
 
   const staticTypes = await getStaticTypes(playroomConfig);
+  const customEntry = playroomConfig.entry
+    ? relativeResolve(playroomConfig.entry)
+    : require.resolve('./defaultModules/entry');
 
   const ourConfig = {
     mode: options.production ? 'production' : 'development',
     entry: {
-      index: [require.resolve('../src/index.js')],
-      frame: [require.resolve('../src/frame.js')],
-      preview: [require.resolve('../src/preview.js')],
+      index: [customEntry, require.resolve('../src/index.js')],
+      frame: [customEntry, require.resolve('../src/frame.js')],
+      preview: [customEntry, require.resolve('../src/preview.js')],
```

Comment:
My one concern with this is you may not want your global styles (for example) to end up in the playroom editor _and_ the playroom frame/preview.

We need this for Braid's playroom, but not everyone else necessarily will. IMO it should be opt-in to apply the entry to all entrypoints, and default to just the frame and preview. This feature should actively be discouraged as a way to style the playroom editor IMO.

Reply:
I've added an escape hatch in https://github.com/seek-oss/playroom/pull/325/commits/504da1a45531055d99129b541e92b2c5a6982f4a